ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,277 @@
+                        onTextChanged: root.tokenAmountValue = Number(text)
+                    }
+
+                    // Just a filler
+                    Item { Layout.fillHeight: true}
+
+                    StatusButton {
+                        enabled: d.ready
+                        text: qsTr("Add")
+                        height: 44
+                        Layout.alignment: Qt.AlignHCenter
+                        Layout.fillWidth: true
+                        onClicked: { root.addToken(root.tokenAmountValue.toString() + " " + root.tokenName, root.tokenImage, root.operator) }
+                    }
+                } // End of Tokens Layout definition
+
+                // TODO
+                Item {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
+
+                // TODO
+                Item {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
+            }
+        }
+    }
+
+    Component {
+        id: operatorsSelectorView
+        ColumnLayout {
+            anchors.fill: parent
+            anchors.margins: 8
+            StatusPickerButton {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 36
+                horizontalPadding: 12
+                spacing: 10
+                bgColor: Theme.palette.primaryColor3
+                contentColor: Theme.palette.primaryColor1
+                image.source: Style.svg("add")
+                text: qsTr("And...")
+                image.height: 12
+                image.width: 12
+                font.pixelSize: 13
+                onClicked: {
+                    root.operator = SQ.Utils.Operators.And
+                    loader.sourceComponent = tabsView
+                }
+            }
+            StatusPickerButton {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 36
+                horizontalPadding: 12
+                spacing: 10
+                bgColor: Theme.palette.primaryColor3
+                contentColor: Theme.palette.primaryColor1
+                image.source: Style.svg("condition-Or")
+                image.height: 12
+                image.width: 12
+                text: qsTr("Or...")
+                font.pixelSize: 13
+                onClicked: {
+                    root.operator = SQ.Utils.Operators.Or
+                    loader.sourceComponent = tabsView
+                }
+            }
+        }
+    }
+
+    Component {
+        id: tokensExtendedView
+
+        // TODO: It probabily will be a reusable component for collectibles and channels
+        TokensListDropdownContent {
+            anchors.fill: parent
+            anchors.topMargin: 8
+            anchors.bottomMargin: 8
+            headerModel: ListModel {
+                ListElement { index: 0; icon: "next"; iconSize: 12; description: qsTr("Back"); rotation: 180; spacing: 0 }
```

Comment:
We do have a `back` icon now

Reply:
Let me keep it as it is and will ask to designers and apply consistency in next tasks as it is only a part of a big functionality and still some pending design details to check. 

Not sure to change it direclty bc in the design it should be like this: 
![Screenshot 2022-08-16 at 12 27 46](https://user-images.githubusercontent.com/97019400/184858393-f77498e6-8e7c-48ab-b088-22cf78feffc2.png)

And our back button looks like this:
![Screenshot 2022-08-16 at 12 28 25](https://user-images.githubusercontent.com/97019400/184858445-62fd2be7-dd73-42d0-a75b-912b74c72536.png)

Reply:
Ah I see, then you just need to exchange the icon, not a biggie